silx/gui/plot/items/histogram.py

```diff
@@ -297,6 +301,20 @@ def setData(self, histogram, edges, align='center', copy=True):
 
         self._updated(ItemChangedType.DATA)
 
+    def setBaseline(self, baseline):
+        if isinstance(baseline, numpy.ndarray):
+            self._baseline = numpy.empty(baseline.shape[0] * 2)
+            for i_value, value in enumerate(baseline):
+                self._baseline[i_value*2:i_value*2+2] = value
```

Comment:
Not sure I get this.
To me the user should input as many baseline points as bins (and retrieve the same from `getBaseline`).
That you have to twice the number of point for rendering should not be visible from the user.